### DISCUSSION TITLE
How do I install/use SQLite with this tool?

### DISCUSSION THREAD

**Discussion by Imaginativeone** [cawa-93/vite-electron-builder#907] [Q&A]
Does anyone have a guide for utilizing SQLite with this tool?

**Comment by TheAutomaTom** [comment]
I got mine working, @Imaginativeone ...
  I used `better-sqlite3`.  _Notice you have to install the types and add a flag to the Main Package's `tsconfig`_

  On Main package, if no database yet exists, it builds a new one and seeds it.  It's going to be created in your project's root directory.  I have not tried to deploy this, yet, but I got errors when I attempted to set a directory (did not try very hard, tbh).  
  At all start ups, it reads that table and sends all rows to the renderer as object[].  I stuck mine into a Pinia store.

Here's the branch, I broke the commits into steps...
[https://github.com/TheAutomaTom/Cabul8r-Electron/pull/10](https://github.com/TheAutomaTom/Cabul8r-Electron/pull/10)
✌️
